docs/decisions/0015-s3gw-ha-model.md

```diff
@@ -0,0 +1,31 @@
+# s3gw ha model
+
+## Context and Problem Statement
+
+We analyzed some High Availability - HA - concepts applied to s3gw when used with Longhorn.
+The final aim of the research is to identify an HA model we can use with the s3gw.
+
+The full HA research work conducted until now has been proposed with the
+[High Availability research](https://github.com/aquarist-labs/s3gw/pull/685) Pull request.
```

Comment:
either "Pull Request" or "pull request". I'm partial towards the latter.

Reply:
For the time being I will go with all downcase; anyway, I'm not sure we should point the PR; perhaps better link the HA document once merged?